src/GraphBLASExt/GxB_Delete.c

```diff
@@ -0,0 +1,22 @@
+#include "./GxB_Delete.h"
+
+GrB_Info GxB_Matrix_Delete
+(
+    GrB_Matrix M,
+    GrB_Index row,
+    GrB_Index col
+)
+{
+    GrB_Matrix Z ;  // 1X1 empty matrix.
+    GrB_Matrix_new (&Z, GrB_BOOL, 1, 1) ;
```

Comment:
I wouldn't mind making this a static global matrix; it would cut down on allocations. If not, Z should be freed in this function. (Also, we should definitely migrate our other functions to invoke this soon.)

Reply:
I though about turning this into either a global variable or a static global,
when dealing with static global, because we're multi threaded and the initialisation process isn't a simple assignment then this might require a lock which I don't want to introduce,
turning this into a global variable (extern) breaks the design you don't see graphblas object relaying on data which gets initialised outside of graphblas, and so unless this causes a performance bottlenecks then I'll just allocate and deallocate this matrix over and over again,
unless Tim can provide us with a set of useful static matrices as part of GraphBLAS...